deploy/charts/harvester/README.md

```diff
@@ -1,108 +1,58 @@
 
-- Use other storage drivers.
-    
-    Currently, storage drivers other than Longhorn are not supported.
+Once a `ManagedChart` is created, Fleet generates a `Bundle` resource to trigger Helm and install the chart. You can learn more about this process in the [Fleet documentation](https://fleet.rancher.io/concepts).
 
-### Configuration
+During upgrades, Harvester runs a script to patch the release version in the `ManagedChart`. This happens in the [`upgrade_manifests.sh`](https://github.com/harvester/harvester/blob/50da36ac3b751c1a1dbfc8d25e5499a4c6216450/package/upgrade/upgrade_manifests.sh#L885-L935) script.
 
-Specify each parameter using the `--set key=value[,key=value]` argument to `helm install` or `helm upgrade`.
+> If you're interested in this change, please check [this issue](https://github.com/harvester/harvester/issues/8163) for more details.
 
-For details on using parameters, please refer to [values.yaml](values.yaml).
+## For Developers
 
-#### Configure KubeVirt Operator
+### How to bump dependency charts
 
-To configure the KubeVirt Operator, you need to know its [parameters](dependency_charts/kubevirt-operator/values.yaml) and put all items under `kubevirt-operator` domain.
+1. Make sure the correct dependency version is specified in `go.mod`.
+2. Run `go mod vendor`.
+3. Update the `Chart.yaml` and `values.yaml` files with the correct version numbers.
+4. In the `deploy/charts/harvester` directory, run `helm dependency update .`
+   
+If the dependency doesn’t involve a change in go.mod, you can start from step 3.
 
-For example, if you want to change the Secret name of admission webhooks, you can do as below.
+Here are example PRs for reference:
 
-```bash
-$ helm install harvester harvester --namespace harvester-system \
-    --set-string kubevirt-operator.containers.operator.certificates.admissionWebhook.secretName=mysecret
-```
-
-If you don't want to install KubeVirt Operator, you can do the following.
+- https://github.com/harvester/harvester/pull/8233
+- https://github.com/harvester/harvester/pull/8210
 
-```bash
-$ helm install harvester harvester --namespace harvester-system \
-    --set kubevirt-operator.enabled=false
-```
+### Use case
 
-#### Configure KubeVirt (CRD resource)
-
-To configure the KubeVirt CRD resource, you need to know its [parameters](dependency_charts/kubevirt/values.yaml) and put all items under `kubevirt` domain.
-
-> **It is worth noting that almost all KubeVirt parameters are string type, including bool field and numeric field.**
-
-For example, if you want to enable the emulation mode on the non-KVM supported hosts, you can do as below.
-
-```bash
-$ helm install harvester harvester --namespace harvester-system \
-    --set-string kubevirt.spec.configuration.developerConfiguration.useEmulation=true
-```
+Harvester occasionally uses Longhorn release candidates. In such cases, download the Longhorn charts and place them in `deploy/charts/harvester/dependency_charts/longhorn`.
```

Comment:
Would we do this for an official release, or mainly for testing? If the latter, can we say something like,

"To use Longhorn release candidates for testing purposes, download the Longhorn charts....."